app/controllers/v0/profile/personal_informations_controller.rb

```diff
@@ -21,7 +21,7 @@ class PersonalInformationsController < ApplicationController
       #   }
       #
       def show
-        response = MPIData.for_user(@current_user).profile
+        response = @current_user.mpi.profile
```

Comment:
If we're going to expose `User.mpi.profile` to places outside of the User model do we want to think about making the `mpi_profile` method public and relying on it as the standard way to directly access a User's MPI profile?

Reply:
That's a good point, although I wonder if making `mpi_profile` public, we should finally make the push to make `mpi` off the `User` class private. I'm thinking of keeping this for this PR, but adding a story to make `User.mpi` private